yarn.lock

```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
+# yarn lockfile v1
+
```

Comment:
- [ ] according to the requirements you should Use `npm init -y` command to create a package.json file. something seems to have gone wrong as there is no package.json file.